<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Terence S. Crofts</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
    <link href='https://fonts.googleapis.com/css?family=Roboto:300,400,700' rel='stylesheet' type='text/css'>
    <link href="style.css" type="text/css" rel="stylesheet">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Questrial&display=swap" rel="stylesheet">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500&display=swap" rel="stylesheet">
  </head>
  <body>

    <!-- Fullscreen hamburger navigation -->
    <input id="toggle" type="checkbox"></input>

    <label for="toggle" class="hamburger">
      <div class="top-bun"></div>
      <div class="meat"></div>
      <div class="bottom-bun"></div>
    </label>
    
    <!-- Navigation links to other pages -->
    <div class="nav">
      <div class="nav-wrapper" id="navigation-width">
        <nav class="flex-col">
          <a href="index.html">Home</a>
          <a href="research.html">Research</a>
          <a href="publications.html">Publications</a>
          <a href="people.html">People</a>
          <a href="lab-news.html">Lab News</a>
          <a href="gallery.html">Gallery</a>
          <a href="contact.html">Contact</a>
          <div class="nav-flex-row">
            <a href="https://twitter.com/terencecrofts?lang=en" target="_blank">
              <img alt="Twitter" src="Resources/images/twitter.png" width=50 height=50>
            </a>
            <a href="https://scholar.google.com/citations?user=3iE1s_wAAAAJ&hl=en" target="_blank">
              <img alt="Google Scholar" src="Resources/images/icons8-google-scholar-50.png" width=50 height=50>
            </a>
            <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Crofts+TS+%5Bauthor%5D" target="_blank">
              <img alt="NCBI Biography" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/07/US-NLM-NCBI-Logo.svg/829px-US-NLM-NCBI-Logo.svg.png" width=50 height=50>
            </a>
          </div> 
        </nav>
      </div>
    </div>

    <!-- Page header -->
    <section class="jumbotron">
      <div class="container">
        <div class="row">
          <h2>Research</h2>
        </div>
      </div>
    </section>

    <!-- Descripton and main image of work section -->
    <section>
      <div>
        <div>
          <p><i>The Crofts' microbiology lab studies the biochemical and genetic factors that underly interactions between emerging contaminants – such as antibiotics and pharmaceuticals – and organisms in microbial communities</i></p>
          </i>
          <img src="Resources/images/220331_CroftsLogo_v2.jpg" class="center"/>
          <p align="center">These interactions impact human health and industry, encompassing antimicrobial resistance and antibiotic degradation as well as changes in the efficacy and toxicity of pharmaceuticals. We develop and use functional metagenomic assays to mine microbiomes for the determinants of these interactions. By studying these functions in isolation and in their native context it is possible to gain greater insight into their evolutionary origins and biochemical niche, as well as to access the great genetic diversity found in microbiomes.</p>
        </div>
      </div>
    </section>

    <!-- First expandable tab button and content -->
    <section>
        <button type="button" class="collapsible"><h4>Antimicrobial Resistance and Catabolism</h4></button>
        <div class="content">
          <h5 align="center">Description</h5>
          <p>Soil and host-associated microbiomes are well-known reservoirs for antimicrobial resistance genes including genes that confer resistance to clinical antibiotics. While many of these genes are well characterized, the tremendous diversity of microbiomes means that many novel and clinically important resistance mechanisms remain to be identified. We have identified several novel variants of streptothricin resistance genes as well as a new mechanism of resistance to this class of antibiotic from the soil microbiome, novel tetracycline and colistin resistance genes in the gut microbiome of wild geese, and have characterized new examples of chloramphenicol reductase enzymes from human pathogens. Discovery and characterization of these mechanisms and their evolvution and regulation can teach us about the ecology of antibiotics in the environment and suggest ways to future-proof new antimicrobials.</p>
          <p>We have also previously discovered and characterized a pathway in multiple soil bacteria to use penicillin as a sole carbon source. The pathway consists of β-lactamase inactivation of the antibiotic, amide hydrolysis of the product to release phenylacetic acid, and breakdown of phenylacetic acid into products that enter central metabolism. We are now investigating catabolism pathways for other antimicrobials besides penicillin. Discovery of enzymes and pathways involved in antibiotic catabolism could lead to new opportunities for semi-synthesis of novel compounds and new approaches to remediating antibiotic pollution in the environment.</p>
          <!-- Copy and paste and fill in necessary links and information of li below to add a publication or press -->
          <h5 align="center">Publications</h5>
          <ul>
            <li>Allman HA, Bernate EP, Franck E, Oliaro FJ, Hartmann EM, Crofts TS (2025) <b>Preparation of functional metagenomic libraries from low biomass samples using METa assembly and their application to capture antibiotic resistance genes</b>. mSystems. <a href="Resources/Paper PDFs/250910_LowbiomassFML.pdf" target="_blank">[PDF]</a></li>

            <li>Franck E, Crofts TS (2024) <b>History of the streptothricin antibiotics and evidence for the neglect of the streptothricin resistome</b>. npj Antimicrobials and Resistance, 2:3. <a href="Resources/Paper PDFs/Franck & Crofts - 2024 - History of the streptothricin antibiotics and evidence for the neglect of the streptothricin resistome.pdf" target="_blank">[PDF]</a></li>
            
            <li>Mullowney MW, Maltseva NI, Endres M, Kim Y, Joachimiak A, Crofts TS,(2022) <b>Functional and structural characterization of diverse NfsB chloramphenicol reductase enzymes from human pathogens</b>. Microbiology Spectrum, 10 (2). e00139-22. <a href="Resources/Paper PDFs/spectrum.00139-22.pdf" target="_blank">[PDF]</a></li>

            <li>Crofts TS,  McFarland AG, Hartmann EM (2021) <b>Mosaic Ends Tagmentation (METa) assembly for extremely efficient construction of functional metagenomic libraries</b>b>. mSystems, 6 (3). e00524-21. <a href="Resources/Paper PDFs/Crofts_METa_2021.pdf" target="_blank">[PDF]</a></li>

            <li>Crofts TS, Sontha P, King AO, Wang B, Biddy BA, Zanolli N, Gaumnitz J, Dantas G (2019) <b>Discovery and characterization of a nitroreductase capable of conferring bacterial resistance to chloramphenicol</b>. Cell Chemical Biology. 26 (4). 559-70.<a href="Resources/Paper PDFs/Crofts et al. - 2019 - Discovery and characterization.pdf" target="_blank">[PDF]</a></li>

            <li>Crofts TS, Wang B, Spivak A, Gianoulis TA, Forsberg KJ, Gibson MK, Johnsky LA, Broomall SM, Rosenzweig CN, Skowronski EW, Gibbons HS, Sommer MOA, Dantas G (2018) <b>Shared strategies for β-lactam catabolism in the soil microbiome</b>. Nature Chemical Biology, 14 (6). 556-64.<a href="Resources/Paper PDFs/Crofts et al. - 2018 - Shared strategies for β-lactam catabolism in the soil microbiome.pdf" target="_blank">[PDF]</a></li>

            <li>Crofts TS, Wang B, Spivak A, Gianoulis TA, Forsberg KJ, Gibson MK, Johnsky LA, Broomall SM, Rosenzweig CN, Skowronski EW, Gibbons HS, Sommer MOA, Dantas G (2017) <b>Draft genome sequences of three β-lactam-catabolizing soil Proteobacteria</b>. Genome Announcements, 5 (32). 8-10.<a href="Resources/Paper PDFs/Crofts et al. - 2017 - Draft Genome Sequences of Three β-Lactam-Catabolizing Soil Proteobacteria.pdf" target="_blank">[PDF]</a></li>

            <li>Crofts TS, Gasparrini AJ, Dantas G (2017) <b>Next-generation approaches to understand and combat the antibiotic resistome</b>. Nature Reviews Microbiology, 15 (7), 422-34.<a href="Resources/Paper PDFs/Crofts, Gasparrini, Dantas - 2017 - Next-generation approaches to understand and combat the antibiotic resistome.pdf" target="_blank">[PDF]</a></li>
          </ul>
          <h5 align="center">Press</h5>
          <ul>
            <li>Researchers capture new antibiotic resistance mechanisms with trace amounts of DNA
              <a href="https://emails.illinois.edu/newsletter/61/447679761.html" target="_blank">[ACES News, September 2025 | Lauren Quinn]</a>
            </li>

            <li>Bacteria may be powerful weapon against antibiotic resistance
              <a href="https://theconversation.com/bacteria-may-be-powerful-weapon-against-antibiotic-resistance-95750" target="_blank">[The Conversation, May 2018 | Terence Crofts]</a>
            </li>

            <li>These Bacteria Can Help Fight Antibiotic Resistance
              <a href="https://www.sciencefriday.com/segments/these-bacteria-can-help-fight-antibiotic-resistance/" target="_blank">[Science Friday, May 2018 | Katie Hiler]</a>
            </li>

            <li>Wash U researchers reveal how some bacteria are able to eat antibiotics
              <a href="https://news.stlpublicradio.org/health-science-environment/2018-05-07/wash-u-researchers-reveal-how-some-bacteria-are-able-to-eat-antibiotics" target="_blank">[St. Louis Public Radio, May 2018 | Shahla Farzan]</a>
            </li>

            <li>Why some bacteria eat antibiotics and what we can do about it
              <a href="https://www.nbcnews.com/health/health-news/why-some-bacteria-eat-antibiotics-what-we-can-do-about-n870181" target="_blank">[NBC news, April 2018 | Maggie Fox]</a>
            </li>

            <li>How penicillin-eating bacteria could help design new antibiotics
              <a href="https://www.abc.net.au/news/science/2018-05-01/antibiotic-eating-bacteria-resistance-agriculture-penicillin/9710388" target="_blank">[ABC news, April 2018 | Belinda Smith]</a>
            </li>

            <li>Bacteria eat antibiotics, and warn each other of threats
              <a href="https://cosmosmagazine.com/biology/bacteria-eat-antibiotics-and-warn-each-other-of-threats/#:~:text=The%20researchers%20found%20that%20bacteria%20exposed%20to%20the%20antibiotic%20tobramcyin,the%20effects%20of%20the%20antibiotic." target="_blank">[Cosmos, April 2018 | Paul Biegler]</a>
            </li>

            <li>How Bacteria Eat Penicillin
              <a href="https://www.the-scientist.com/daily-news/how-bacteria-eat-penicillin-36350" target="_blank">[The Scientist, April 2018 | Shawna Williams]</a>
            </li>

            <li>Bacteria’s appetite may be key to cleaning up antibiotic contamination
              <a href="https://medicine.wustl.edu/news/bacterias-appetite-may-be-key-to-cleaning-up-antibiotic-contaminated-environments/#:~:text=Bacteria's%20appetite%20may%20be%20key%20to%20cleaning%20up%20antibiotic%20contamination,-Antibiotics%20in%20environment&text=Antibiotics%20in%20the%20environment%20contribute%20to%20drug%20resistance.&text=Louis%20have%20figured%20out%20how,antibiotic%2Dcontaminated%20soil%20and%20waterways." target="_blank">[WUSM news release, April 2018 | Tamara Bhandari]</a>
            </li>
        </ul>
         <h5 align="center">Press</h5>
         <ul>
            <li>Researchers capture new antibiotic resistance mechanisms with trace amounts of DNA
              <a href="https://emails.illinois.edu/newsletter/61/447679761.html" target="_blank">[ACES News, September 2025 | Lauren Quinn]</a>
            </li>
        </ul>
      </div>

      <!-- Second expandable content section, if need to edit see above notes, copy and paste examples, fill in resources -->
      <button type="button" class="collapsible"><h4>Pharmaceutical Metabolism</h4></button>
      <div class="content">
        <h5 align="center">Description</h5>
        <p>The human microbiome, and especially the large intestinal microbial community, is an important actor in the efficacy/toxicity of many pharmaceuticals, but the vast majority of these interactions lack description at the enzymatic level. The Crofts lab is approaching this field by using a combination of next-generation sequencing and functional metagenomic techniques to identify and capture genes encoding enzymes with activity against pharmaceuticals. We are investigating the xenobiotic metabolism of nitro-pharmaceuticals by the human microbiome using the antibiotic chloramphenicol as a model compound. We have structurally and functionally characterized enzymes from host-associated bacteria that modify chloramphenicol at the nitro position and are looking to expand this research further. Other important lines of research into microbiome-pharmaceutical interactions are how microbiome-immune system interactions alter the efficacy of immunotherapeutics, and how early-life antibiotic therapy might also alter the developing infant microbiome.</p>
        <h5 align="center">Publications</h5>
        <ul>
          <li>Mullowney MW, Maltseva NI, Endres M, Kim Y, Joachimiak A, Crofts TS(2022) <b></b>Functional and structural characterization of diverse NfsB chloramphenicol reductase enzymes from human pathogens</b>. Microbiology Spectrum, 10 (2). e00139-22. <a href="Resources/Paper PDFs/spectrum.00139-22.pdf" target="_blank">[PDF]</a></li>

          <li>Crofts TS, Sontha P, King AO, Wang B, Biddy BA, Zanolli N, Gaumnitz J, Dantas G (2019) <b>Discovery and characterization of a nitroreductase capable of conferring bacterial resistance to chloramphenicol</b>. Cell Chemical Biology. 26 (4). 559-70.<a href="Resources/Paper PDFs/Crofts et al. - 2019 - Discovery and characterization.pdf" target="_blank">[PDF]</a></li>

          <li>Keen EC*, Crofts TS, Dantas G (2018) <b>Checkpoint checkmate: Microbiota modulation of cancer immunotherapy</b>. Clinical Chemistry, 64 (9). 1280-3.<a href="Resources/Paper PDFs/Keen, Crofts, Dantas - 2018 - Checkpoint Checkmate Microbiota Modulation of Cancer Immunotherapy.pdf" target="_blank">[PDF]</a></li>

          <li>Gasparrini AJ*, Crofts TS, Gibson MK, Tarr PI, Warner BB, Dantas G (2016) <b>Antibiotic perturbation of the preterm infant gut microbiome and resistome</b>. Gut Microbes, 7 (5), 443-9.<a href="Resources/Paper PDFs/Gasparrini et al. - 2016 - Antibiotic perturbation of the preterm infant gut microbiome and resistome.pdf" target="_blank">[PDF]</a></li>

          <li>Gibson MK*, Crofts TS, Dantas G (2015) <b>Antibiotics and the developing infant gut microbiota and resistome</b>. Current Opinion in Microbiology, 27, 51-6.<<a href="Resources/Paper PDFs/Gibson, Crofts, Dantas - 2015 - Antibiotics and the developing infant gut microbiota and resistome.pdf" target="_blank">[PDF]</a></li>
        </ul>
      </div>

      <!-- Third expandable content section, if need to edit see above notes, copy and paste examples, fill in resources -->
      <button type="button" class="collapsible"><h4>Functional Metagenomics</h4></button>
      <div class="content">
        <h5 align="center">Description</h5>
        <p>The Crofts lab is also involved in the development of high-throughput methods for studying microbiomes at the genetic and biochemical level. One powerful tool for this is functional metagenomic libraries which allow sequence-naïve and cultivation-independent gene discovery. Functional metagenomic libraries can be used to discover novel genes for antimicrobial resistance or pharmaceutical modification (among other functions) from microbiomes. We have invented and applied a new method for creating functional metagenomic libraries that consistently produces significantly larger libraries using a fraction of the quantity of input DNA needed by current methods. We are in the process of expanding and applying this method further. The ability to use less DNA more efficiently will allow functional metagenomic libraries to be prepared from rare and low biomass microbiomes that were previously out of reach of this method. </p>
        <h5 align="center">Publications</h5>
        <ul>
          <li>Allman HA, Bernate EP, Franck E, Oliaro FJ, Hartmann EM, Crofts TS (2025) <b></b>Preparation of functional metagenomic libraries from low biomass samples using METa assembly and their application to capture antibiotic resistance genes</b>. mSystems. <a href="Resources/Paper PDFs/250910_LowbiomassFML.pdf" target="_blank">[PDF]</a></li>

          <li>Crofts TS,  McFarland AG, Hartmann EM (2021) <b></b>Mosaic Ends Tagmentation (METa) assembly for highly efficient construction of functional metagenomic libraries</b>. mSystems, 6 (3). e00524-21. <a href="Resources/Paper PDFs/Crofts_METa_2021.pdf" target="_blank">[PDF]</a></li>

          <li>Crofts TS, Gasparrini AJ*, Dantas G (2017) <b>Next-generation approaches to understand and combat the antibiotic resistome</b>. Nature Reviews Microbiology, 15 (7), 422-34.<a href="Resources/Paper PDFs/Crofts, Gasparrini, Dantas - 2017 - Next-generation approaches to understand and combat the antibiotic resistome.pdf" target="_blank">[PDF]</a></li>
        </ul>
      </div>

      <script>
        // This is the methods used to expand and collapse sections, you should not need to edit
        var coll = document.getElementsByClassName("collapsible");
        var i;

        for (i = 0; i < coll.length; i++) {
        coll[i].addEventListener("click", function() {
          this.classList.toggle("active");
          var content = this.nextElementSibling;
          if (content.style.display === "block") {
            content.style.display = "none";
          } else {
            content.style.display = "block";
          }
        });
        }

      </script>
    </section>
  </body>
</html>
